doc-changelog/action.yml

```diff
@@ -160,23 +160,39 @@ runs:
           rm $file
         fi
 
+    - name: "Clean PR title"
+      env:
+        PR_TITLE: ${{ github.event.pull_request.title }}
+      shell: python
+      run: |
+        import os
+
+        # Remove extra whitespace
+        clean_title = " ".join("""${{ env.PR_TITLE }}""".split())
```

Comment:
```suggestion
        # Retrieve title
        clean_title = os.getenv('PR_TITLE')

        # Remove extra whitespace
        clean_title = clean_title.strip()
```

Reply:
@RobPasMue does strip also remove extra whitespace between words, or only remove trailing whitespace?

Reply:
Strip works for leading and trailing whitespaces which is what we should focus on... if users add extra whitespaces inside the PR title, this might be intended or it might not... but I don't think we should be "playing" with the "internals" of the PR title.

Reply:
Imagine the following:

```
docs: adding `   ` instead of ` ` inside the API ref landing page
```

This would get translated into:
```
docs: adding ` ` instead of ` ` inside the API ref landing page
```

Which... is definitely wrong. We want it to show up as the first one. My point is, if the user made a mistake with the number of whitespaces inside the PR title itself, then so be it. But we shouldn't be assuming it.